ansible/group_vars/.gitignore

```diff
@@ -0,0 +1,19 @@
+#-- vim
```

Comment:
I think this file can be removed. Maybe the contents can be moved to the gitignore file in the root